### PR TITLE
Add lucalanziani to Build WG Members list

### PR DESCRIPTION
**Pull request by LucaLanziani**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ missing please open an issue. If you are interested in joining check out
 - [@rvagg](https://github.com/rvagg) - Rod Vagg
 - [@Starefossen](https://github.com/Starefossen) - Hans Kristian Flaatten
 - [@Trott](https://github.com/Trott) - Rich Trott
+- [@lucalanziani](https://github.com/lucalanziani) - Luca Lanziani
```
Comment (maclover7) [inline README.md]: GitHub usernames are sorted alphabetically :)

Comment (LucaLanziani) [reply README.md]: ah, I've checked if the order was on first names, didn't think of checking the github usernames, fixing that.

**Comment by LucaLanziani** [comment]
@maclover7 updated

**Comment by rvagg** [comment]
neato, welcome!

**Comment by maclover7** [comment]
@LucaLanziani This has plenty of approvals, feel free to land at will :)

**Pull request merged**